### PR TITLE
Add seperate worker instance and task queue

### PR DESCRIPTION
**Pull request by banesullivan**
cc @zachmullen for #285 

Here are some experimental changes I had for adding a separate worker instance and task queue but it isn't quite working

### CODE REVIEW

**Comment by zachmullen** [comment]
What's the problem? (FYI @brianhelba )

**Comment by banesullivan** [comment]
The problem is tasks are launching or being received... I imagine this is a trivial fix for something I am forgetting to set

File: rgd/celery.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 # Using a string config_source means the worker doesn't have to serialize
 # the configuration object to child processes.
 app = Celery('rgd', config_source='django.conf:settings', namespace='CELERY')
+app.conf.task_default_queue = 'default'
```
Comment (brianhelba) [inline rgd/celery.py:14]: All the other Celery config is done via Django settings. I think you should set that there, for consistency.

File: docker-compose.override.yml [hunk 2]
```diff
@@ -26,7 +26,34 @@ services:
       "--app", "rgd.celery",
       "worker",
       "--loglevel", "INFO",
-      "--without-heartbeat"
+      "--without-heartbeat",
+      "-Q default"
```
Comment (brianhelba) [inline docker-compose.override.yml:30]: You could use `-X` / `--exclude-queues` instead: https://docs.celeryproject.org/en/stable/reference/cli.html#cmdoption-celery-worker-X

File: rgd/celery.py (hunk 1 above)
Comment (banesullivan) [reply rgd/celery.py]: With `--exclude-queues` I don't think this queue is needed.

**Comment by banesullivan** [comment]
@brianhelba @zachmullen, I cannot get the tasks with `queue='priority'` to be received by the `celery_priority` container. All other tasks are launching in the default `celery` container though

**Comment by banesullivan** [comment]
Closing due to inactivity and other priorites

**Pull request closed without merging**